app/src/main/res/values-sv/strings.xml

```diff
@@ -63,7 +63,7 @@
     <string name="session_details_section_title_links">Länkar</string>
     <string name="choose_alarm_time">Välj alarm tid</string>
     <string name="reminders">Alarm</string>
-    <string name="usage">Tips: Håll inne på en lektion för att sätta ett alarm.</string>
+    <string name="usage">Tips: Håll inne på en föreläsning för att sätta ett alarm.</string>
```

Comment:
Hint: Please take the word "lecture" not so strict. At a conference there can be lectures but also workshops or other gatherings. If you can think of a word that fits all of them feel free to change it here and in other strings.

Reply:
I think that "föreläsning" is the best one I can think of now.